pkg/csi_driver/node.go

```diff
@@ -79,16 +80,31 @@ func (s *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublish
 	// Mount source
 	source := fmt.Sprintf("%s:/%s", attr[attrIp], attr[attrVolume])
 
+	// FileSystem type
+	fstype := "nfs"
+
 	// Mount options
 	options := []string{}
+
+	// Windows specific values
+	if runtime.GOOS == "windows" {
```

Comment:
nit: create a util function `IsWindows` or something

Reply:
This is how kubernetes/kubernetes checks for Windows in many [locations](https://github.com/kubernetes/kubernetes/search?q=GOOS+windows&unscoped_q=GOOS+windows). I don't think it's worth adding a wrapper.